include/mbedtls/md.h

```diff
@@ -132,220 +138,255 @@ void mbedtls_md_free( mbedtls_md_context_t *ctx );
 #define MBEDTLS_DEPRECATED
 #endif
 /**
- * \brief           Select MD to use and allocate internal structures.
- *                  Should be called after mbedtls_md_init() or mbedtls_md_free().
+ * \brief           This function selects the MD to use, and allocates internal 
+ *                  structures.
+ *
+ *                  It should be called after mbedtls_md_init() or mbedtls_md_free().
  *                  Makes it necessary to call mbedtls_md_free() later.
  *
  * \deprecated      Superseded by mbedtls_md_setup() in 2.0.0
  *
- * \param ctx       Context to set up.
- * \param md_info   Message digest to use.
+ * \param ctx       The context to set up.
+ * \param md_info   The message digest to use.
```

Comment:
Suggestion: `The information structure of the message-digest to use`. I'd emphasize the `information structure` part to separate it from the numeric identifiers in `mbedtls_md_type_t`. In any case, a hyphen should be added.

Reply:
fixed in all descriptions of the md_info param:
"The information structure of the message-digest algorithm to use."